### PR TITLE
Alert component pull request. 

### PR DESCRIPTION
**Pull request by SanjayJatti**
alert component of component librar
alert component
y

### CODE REVIEW

File: Componets/Alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    border-left: 5px solid var(--dark-success-color);
+    
+}
+.alert-warning{
+    color: var(--dark-warning-color) ;
+    background-color: var(--light-warning-color);
+    border-left: 5px solid var(--dark-warning-color);
+}
+.alert-danger{
+    color: var(--dark-danger-color);;
```
Comment (amarphule) [inline Componets/Alert/alert.css]: double semicolon please check

**Review by amarphule — COMMENTED** [review]
there is one typo mistake other than that it look wornderful, great work

**Review by omgaikwad — COMMENTED** [review]
Only one typo error of extra semi-colon, otherwise everything looks good, great work!

**Comment by SanjayJatti** [comment]
> Only one typo error of extra semi-colon, otherwise everything looks good, great work!

Fixed!

**Comment by vaibhav18matere** [comment]
good work, good to go !

**Pull request merged**